packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/tests/index.test.js

```diff
@@ -259,4 +278,37 @@ describe('BlocksEditor toolbar', () => {
 
     expect(screen.getByText(title)).toBeInTheDocument();
   });
+
+  it('check if a mixed selected content show only one option selected in the dropdown', async () => {
+    setup(mixedInitialValue);
+
+    const headingsDropdown = screen.getByRole('combobox', { name: /Select a block/i });
+
+    // Set the selection to cover the entire content
+    waitFor(() => {
```

Comment:
why are you not awaiting?

Reply:
With await I have the console errors when I run the test asking me to wrap the instruction inside an act

Reply:
it sounds like not awaiting is silencing the error but not really fixing it then, as far as I know waitFor is meant to be awaited to actually work

Reply:
^^^ exactly what @remidej said, @simotae14 can you fix your tests correctly please :)